File: C++/day13.cpp
//Write MyBook class
class MyBook:public Book
    {   int price;
        public:
        MyBook(string t,string a,int p):Book(t,a){price=p;}
        void display()
            {   cout<<"Title:\t"<<title; cout<<"\n";
                cout<<"Author:\t"<<author; cout<<"\n";
                cout<<"Price:\t"<<price<<"\n";
                
            }
            
    };

File: PYTHON2/day13.py
#Write MyBook class
class MyBook(Book):
    def __init__(self, title, author, price):
        self.price = price
        super(MyBook, self).__init__(title, author)
    def display(self):
        print "Title:", self.title
        print "Author:", self.author
        print "Price:", self.price
